data_steward/cdr_cleaner/cleaning_rules/clean_height_weight.py

```diff
@@ -590,113 +601,204 @@
+
+    def get_query_specs(self):
+        """
+        Returns a list of dictionary query specifications.
 
-    additional_argument = {
-        parser.SHORT_ARGUMENT: '-n',
-        parser.LONG_ARGUMENT: '--sandbox_dataset_id',
-        parser.ACTION: 'store',
-        parser.DEST: 'sandbox_dataset_id',
-        parser.HELP: 'Please specify the sandbox_dataset_id',
-        parser.REQUIRED: True
-    }
-    args = parser.default_parse_args([additional_argument])
-    return args
+        :return: A list of dictionaries. Each dictionary contains a single query
+            and a specification for how to execute that query. The specifications
+            are optional but the query is required.
+        """
+
+        # height
+        save_height_table_query = {
+            cdr_consts.QUERY:
+                CREATE_HEIGHT_SANDBOX_QUERY.render(
+                    project_id=self.get_project_id(),
+                    dataset_id=self.get_dataset_id(),
+                    sandbox_dataset_id=self.get_sandbox_dataset_id(),
+                    height_table=HEIGHT_TABLE),
+            cdr_consts.DESTINATION_TABLE:
+                HEIGHT_TABLE,
+            cdr_consts.DESTINATION_DATASET:
+                self.get_sandbox_dataset_id(),
+            cdr_consts.DISPOSITION:
+                WRITE_TRUNCATE
+        }
+
+        save_new_height_rows_query = {
+            cdr_consts.QUERY:
+                NEW_HEIGHT_ROWS_QUERY.render(
+                    project_id=self.get_project_id(),
+                    dataset_id=self.get_dataset_id(),
+                    sandbox_dataset_id=self.get_sandbox_dataset_id(),
+                    new_height_rows=NEW_HEIGHT_ROWS,
+                    height_table=HEIGHT_TABLE),
+            cdr_consts.DESTINATION_TABLE:
+                NEW_HEIGHT_ROWS,
+            cdr_consts.DESTINATION_DATASET:
+                self.get_sandbox_dataset_id(),
+            cdr_consts.DISPOSITION:
+                WRITE_TRUNCATE
+        }
+
+        delete_height_rows_query = {
+            cdr_consts.QUERY:
+                DELETE_HEIGHT_ROWS_QUERY.render(
+                    project_id=self.get_project_id(),
+                    dataset_id=self.get_dataset_id()),
+            cdr_consts.DESTINATION_TABLE:
+                MEASUREMENT,
+            cdr_consts.DESTINATION_DATASET:
+                self.get_dataset_id(),
+            cdr_consts.DISPOSITION:
+                WRITE_TRUNCATE
+        }
+
+        insert_new_height_rows_query = {
+            cdr_consts.QUERY:
+                INSERT_NEW_ROWS_QUERY.render(project_id=self.get_project_id(),
+                                             dataset_id=self.get_dataset_id(),
+                                             new_rows=NEW_HEIGHT_ROWS),
+            cdr_consts.DESTINATION_TABLE:
+                HEIGHT_TABLE,
+            cdr_consts.DESTINATION_DATASET:
+                self.get_sandbox_dataset_id(),
```

Comment:
I think this needs to be the measurement table
```suggestion
            cdr_consts.DESTINATION_TABLE:
                MEASUREMENT,
            cdr_consts.DESTINATION_DATASET:
                self.get_dataset_id(),
```